service-loadbalancer/service_loadbalancer.go

```diff
@@ -137,6 +140,30 @@ type loadBalancerConfig struct {
 	startSyslog bool   `description:"indicates if the load balancer uses syslog."`
 }
 
+type staticPageHandler struct {
+	pagePath     string
+	pageContents []byte
+}
+
+// Get serves the page stored by load
```

Comment:
since you don't have a function called load, please modify comment or add a load function and do the readfile from there instead of newStaticPageHandler